src/migration/1743776947033-UserTransactionValueFee.ts

```diff
@@ -0,0 +1,19 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class UserTransactionValueFee1743776947033
+  implements MigrationInterface
+{
+  name = 'UserTransactionValueFee1743776947033';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TABLE "user_transaction" ADD COLUMN IF NOT EXISTS "valueIncFees" integer NOT NULL`,
```

Comment:
Will manually do alter so I can set existing values appropriately. Mostly to avoid doing migration in two steps:
1. Column with default value 0
2. Update values
3. Then remove default from column